cmd/spire-agent/cli/command/run.go

```diff
@@ -102,6 +103,17 @@ func (*RunCommand) Synopsis() string {
 func parseFile(filePath string) (*CmdConfig, error) {
```

Comment:
Could we have the functions that are exactly the same in the agent and in the server shared from a common package?
That would avoid having to maintain both when we need changes.